prometheus/base/alerting/prometheus.pod_alerts.yaml

```diff
@@ -25,3 +25,14 @@ spec:
       annotations:
         summary: "Pod {{ $labels.pod }} is crash looping"
         description: "Pod {{ $labels.namespace }}/{{ $labels.pod }} ({{ $labels.container }}) is in waiting state (reason: 'CrashLoopBackOff') for more than 15 minutes."
+    - alert: QuotaExceeded
+      expr: |
+        kube_resourcequota{type="used", namespace!~"(.*-tenant|openshift-.*|kube-.*|default)"} /
+        ignoring (instance, job, type)
+        (kube_resourcequota{type="hard", namespace!~"(.*-tenant|openshift-.*|kube-.*|default)"} > 0) > 1
```

Comment:
@amisstea is this ever going to be higher than one?

Reply:
Yes. I experimented by creating a `Pod` and then a `ResourceQuota` with limits and requests lower than what was defined in the `Pod`.

![image](https://user-images.githubusercontent.com/22659204/233690181-b2de516d-e386-40e7-8e61-82b7796bd60e.png)

Reply:
Cool

Reply:
On second thought, suppose that we consider the more standard flow in which a quota is created before the pod, will it ever alert? So maybe it should be higher or equal instead?

and also, is it possible that the quota will be 98% but resources cannot be created, but the will not fill the quota?
Suppose you have 10GB mem quota and its 9GB taken, and you try to create 2GB pods, will the quota reach 100%? Should we set the value to something smaller than 100%?

Reply:
Also, if we do want to fire alerts for high quota usage, I think it should be defined as a separate alert.